django/api/services/tests/test_calculate_rebate.py

```diff
@@ -0,0 +1,177 @@
+from unittest import TestCase
+from rest_framework.serializers import ValidationError
+from django.test import TestCase
+from ...models.go_electric_rebate_application import GoElectricRebateApplication
+from ...models.household_member import HouseholdMember
+from django.contrib.auth import get_user_model
+from ..calculate_rebate import calculate_rebate_amount
+
+
+class TestCalculate(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        user = get_user_model()
```

Comment:
Would uppercase this as a classname convention

Reply:
No it doesn't

User = get_user_model()

Reply:
oh i thought you were referring about the function that was defined on the first line of that.. okay yep the user is fixed